.github/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8">
+    <title>판다마켓</title>
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <div class="all">
+        <div class="title-head">
+          <a class="login-link" href="/">
+            <img class="title-logo" src="img/rogo1.png">
+          </a>
+          <button class="login">
+            <a class="login-link" href="/login">로그인</a>
+          </button>
```

Comment:
❗️ 수정요청  
로그인 버튼을 눌렀을 때 './login'으로 이동시켜주는 역할을 하니, 앵커 태그만 남겨두셔도 될 것 같습니다~  
버튼 요소처럼 보여도 꼭 버튼 태그로 작성하실 필요는 없습니다!  
해당 태그의 역할에 따라 태그를 작성하시는 것이 더 시맨틱하게 작성하는 방법입니다.

```suggestion
          <a class="login-link" href="/login">로그인</a>
```